bin/aapp_dr_runner.py

```diff
@@ -832,7 +823,9 @@ def setup_aapp_processing(config):
     aapp_satellite_list = os.getenv('PAR_NAVIGATION_DEFAULT_LISTESAT').split()
     if config['platform_name'] not in aapp_satellite_list:
         LOG.warning(
-            "Can not find this platform in AAPP config variable PAR_NAVIGATION_DEFAULT_LISTESAT. Will try to find matches. But it can be a good idea to change this variable in the ATOVS_ENV7 file.")
+            "Can not find this platform in AAPP config variable PAR_NAVIGATION_DEFAULT_LISTESAT. " +
+            "Will try to find matches. But it can be a good idea to change this " +
```

Comment:
```suggestion
            "Can not find this platform in AAPP config variable PAR_NAVIGATION_DEFAULT_LISTESAT. "
            "Will try to find matches. But it can be a good idea to change this "
```

you don't need `+`
```python
In [14]: ('aaaa'
    ...: 'bbbb'
    ...: 'cccc')
Out[14]: 'aaaabbbbcccc'
```